test_conformance/basic/test_async_copy3D.cpp

```diff
@@ -409,14 +391,12 @@ int test_copy3D(cl_device_id deviceID, cl_context context,
         if (i < (int)(globalWorkgroupSize * planesCopiesPerWorkItem - 1)
                 * elementSize)
         {
-            int outIdx = i
-                    * (numLines * numElementsPerLine + numLines * dstLineStride
-                       + dstPlaneStride)
+            int outIdx = i * dstPlaneStride
                 + (numLines * elementSize) * (numElementsPerLine)
-                + (numLines * elementSize) * (dstLineStride);
+                + (numLines * elementSize) * (dstLineMargin);
```

Comment:
Clearer to write
`+ numLines * dstLineStride * elementSize;`
instead of lines 395 and 396?

(It is somewhat confusing that i,j,k already contain the factor of elementSize, irrespective of this patch.)